README.md

```diff
@@ -2,7 +2,7 @@
 
 # sparkmagic
 
-Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](https://github.com/cloudera/hue/tree/master/apps/spark/java), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
+Sparkmagic is a set of tools for interactively working with remote Spark clusters through [Livy](http://livy.incubator.apache.org./), a Spark REST server, in [Jupyter](http://jupyter.org) notebooks.
```

Comment:
I just realized, can you also make it an https link? Sorry for multiple comments.